pymilvus/decorators.py

```diff
@@ -108,3 +109,18 @@ def handler(*args, **kwargs):
                 raise e
         return handler
     return wrapper
+
+def tracing_request():
+    def wrapper(func):
+        @functools.wraps(func)
+        def handler(self, *args, **kwargs):
+            level = kwargs.get("log_level", None)
+            req_id = kwargs.get("client_request_id", None)
```

Comment:
Three question:
- 1, what's the type of `client_request_id`?
- 2, how should users construct the `client_request_id`?
- 3, what's the behavior if two same `client_request_id`s were met in server?

Reply:
1. client_request_id is a user defined string to trace request
2. if two same client_request_id are same, the corresponding requests have same trace id in their logs. It's recommended to use random string as the request id in order to avoid this problem.

Reply:
OK, got it. Better to give a simple documentation to guide users how to use this feature when everything is ready.